### DISCUSSION TITLE
Minio Replication version enabled and timestamp changed Problem

### DISCUSSION THREAD

**Discussion by sathishkumar-p** [minio/minio#17769] [Q&A]
Hello, 
I recently created a replication for my minio server. I observed a few things on replicated serve. 

1. Versioning for the bucket is enabled automatically on the replicated 
2.  file present in the bucket's last modified date is changed it's not the exact same date in the source.

**Comment by klauspost** [comment]
@sathishkumar-p Could you provide some information on your setup? Configuration, versions, replication setup, etc.

**Reply by sathishkumar-p** [reply]
Minio Version
![image](https://github.com/minio/minio/assets/25504642/8f82cfaf-6bc2-4c92-b480-8dca8c1b9f61)
Configuration:
4 servers 4 drivers per server running in the Kubernetes cluster deployed with the help of minio-operator

Replication Setup:
Installed two minio tenant on the same cluster
Tenant A will act as the primary and Tenant B will act as replicated server.

**Reply by klauspost** [reply]
Could you also provide the documentation for the issues you are seeing - you say timestamps are not matching.

Perhaps show an object from both sites with `mc ls --versions ALIAS/....`.

**Reply by ravindk89** [reply]
Can you also confirm that NTP is correctly configured for all hosts involved here?

**Reply by sathishkumar-p** [reply]
@klauspost 
For additional information, We are trying to migrate from bitnami minio version to the official minio. So server-source A(bitnami) I will like to move the bucket, and policy to source B (official minio). So i setup a replication between the server.

**Reply by klauspost** [reply]
@sathishkumar-p How did you set up the replication?

Please provide listings of the unexpected differences you are seeing. So far you are just claiming "timestamp changed", and no additional information. We can't help you if you don't provide information about what the actual problem is.

**Reply by sathishkumar-p** [reply]
@klauspost 
Like i mentioned replication is successful, but only the problem is file timestamp is changed to the time is replicated and the versioning of the bucket is enabled 
Source: 
![image](https://github.com/minio/minio/assets/25504642/77534f37-dcba-4aa4-b6d9-facc81107a45)
Replicated Destination:
![image](https://github.com/minio/minio/assets/25504642/4622aac6-e711-4937-a5a5-7bf1cba43f87)

**Reply by klauspost** [reply]
You still haven't told me how you replicated it.

If you set up *serverside* replication correctly timestamps will be replicated. Any clientside replication is not able to transfer modification time (or version IDs).

**Reply by sathishkumar-p** [reply]
@klauspost 
I have set up a site replication between the minio tenant. Sorry, I am not able to get your point clear. Could you please elaborate a little bit ?
![image](https://github.com/minio/minio/assets/25504642/3cda133a-78fa-456b-a442-2f56a7975b16)

**Reply by sathishkumar-p** [reply]
![image](https://github.com/minio/minio/assets/25504642/134bf181-fb7b-439a-a04c-25482e0a8b29)

**Reply by klauspost** [reply]
@sathishkumar-p Looks fine, The versions you list are more than a year old, though, so a lot of changes has changed since then.

We don't provide support for older versions except through [subscriptions](https://min.io/pricing).

I suggest you upgrade and see if the issue persists even after upgrading on new objects.
